### ISSUE TITLE
Adding comments doesn't work (Sandstorm)

### ISSUE THREAD

**Issue by jparyani** [wekan/wekan#270]
When trying to add a comment, nothing happens when I click the "Comment" button, and the following is printed to the JS console:

```
insert failed: Error: Board is required
    at r (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:80:2236)
    at n (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:80:2108)
    at i.each.a.Collection.(anonymous function) [as insert] (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:80:6137)
    at null.<anonymous> (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:116:3557)
    at s.extendCollectionInstance.i.each.c.(anonymous function) [as insert] (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:116:1517)
    at BlazeComponent.extendComponent.events.submit .js-new-comment-form (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:343:473)
    at https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:225:4884
    at Object.d._withCurrentView (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:46:22533)
    at https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:225:4845
    at e._withTemplateInstanceFunc (https://nt80r1xo4attwsbwaq1r.oasis.sandstorm.io/90599a4f8099473a5b390df3f3d4541185876cdd.js:47:4967)
```

I can reproduce it consistently with the Sandstorm version of Wekan (from https://apps.sandstorm.io/app/m86q05rdvj14yvn78ghaxynqz7u2svw6rnttptxx49g1785cdv1h), but it doesn't occur when running locally from master.

Tested on both Chrome and Firefox stable on OSX.

**Comment by mquandalle** [comment]
This was fixed in 549f8fee3aee3675ae2c04fa14b88f4a783df2c6 but net deployed for sandstorm. I will publish a new spk with more bug fixes soon.

**Comment by mquandalle** [comment]
This is fixed in v0.9-rc2 and released on the Sandstorm market.
